fcrepo-webapp/src/main/webapp/static/constraints/owl2html.xsl

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Okay, that's cool, but let's make a ticket to get some caching or memoization in there.